src/net/neighbors.rs

```diff
@@ -2678,7 +2889,7 @@ mod test {
                 match peer_2.network.walk {
                     Some(ref w) => {
                         assert_eq!(w.result.broken_connections.len(), 0);
-                        assert_eq!(w.result.dead_connections.len(), 0);
+                        // assert_eq!(w.result.dead_connections.len(), 0);
```

Comment:
Are this 2 assertions being disabled permanently?

Reply:
Yes; will remove the commented code.